### PR TITLE
Navigation grid component

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Created Navigation and Grid Component 
Please Review 

Thank you

### CODE REVIEW

**Review by 05him — APPROVED** [review]
image
   
just a small suggestion that , you are already having a class name "flex-centered" so without writing display: flex everywhere you can use that class here and adjust the justify-content only according to your need .. 
  
  else the rest of the code looks good

File: documentation.html [hunk 1]
```diff
@@ -1309,6 +1315,182 @@ <h3 class="component-subheading">Spaced List Group</h3>
...
+
+            <div class="navigation-component-container">
+              <header>
+                <nav class="navbar box-shadow">
+                  <div class="left-navbar">
+                    <p class="logo"><a href="#">Chalk</a></p>
+                  </div>
+                  <ul class="right-navbar">
+                    <li class="NoShow"><a href="#">Home</a></li>
+                    <li class="NoShow"><a href="#">Documentation</a></li>
```
Comment (vrx29) [inline documentation.html]: classname "NoShow" should not be like this. please make it like other classnames that you are using like "no-show"

Comment (vishal804) [reply documentation.html]: Done the Changes

**Comment by vishal804** [comment]
> image
> 
> just a small suggestion that , you are already having a class name "flex-centered" so without writing display: flex everywhere you can use that class here and adjust the justify-content only according to your need ..
> 
> else the rest of the code looks good

Will Do That 
Thank you For Review

File: documentation.html (hunk 1 above)
Comment (vishal804) [reply documentation.html]: Thank You For Review

**Pull request merged**